Sources/Moya/MoyaProvider.swift

```diff
@@ -149,19 +168,23 @@ public extension MoyaProvider {
     // Swift won't let us put the StubBehavior enum inside the provider class, so we'll
     // at least add some class functions to allow easy access to common stubbing closures.
 
+    /// A class method that returns StubBehavior.never.
```

Comment:
Again, I would avoid implementation details and say what does this stub option mean (in that case it wouldn’t stub at all)